packages/one-shot/paas-2925-fix-dumpS_path_typo.yaml

```
@@ -0,0 +1,67 @@
+---
+type: update
+version: 1.5.2
+name: PAAS-2925 - Fix dumpS_path typo and duplication
+id: paas-2925-fix-dumps-path-typo-and-duplication
+description:
+  short: Fix DUMPS_PATH variable name type and its duplication
+
+mixins:
+  - "../../mixins/common.yml"
+  - "../../mixins/jahia.yml"
+  - "../../mixins/haproxy.yml"
+
+globals:
+  toRestart: []
+
+onInstall:
+  - forEach(nodes.proc):
+      - fixDumpPath: ${@i.id}
```

if the issue is that you need the node id, to avoid the useless loop you could do this:
```suggestion
  - fixDumpPath: ${nodes.proc.first.id}
```

---

indeed, cloudscript made me lose my temper
fixed !